bVNC/src/main/java/com/iiordanov/bVNC/dialogs/GetTextFragment.java

```diff
@@ -100,7 +102,7 @@ public void afterTextChanged(Editable arg0) {}
     private String t2;
     private String t3;
     private boolean keepPassword;
-
+    private boolean sw1;
```

Comment:
If we are keeping the switch name, let's make this variable named with a full word.